templates/repo/branch/list.tmpl

```diff
@@ -67,140 +67,136 @@
 			</div>
 		{{end}}
 
-		{{if .Branches}}
```

Comment:
The removed line seems to be a unrelated change. If I understand correctly, this PR's purpose is to "unify search boxes", but not "change the branch render logic" ....

Reply:
The reason is that otherwise the search bar disappears when there are no results....